src/github.com/getlantern/flashlight/pac.go

```diff
@@ -41,16 +44,33 @@ func setUpPacTool() error {
 	return nil
 }
 
+func setProxyAddr(addr string) {
```

Comment:
Oh I mean it just proxy all traffic except localhost, so it no longer depends on proxied sites list. Sorry the confusion caused.